covert/__main__.py

```diff
@@ -99,20 +99,21 @@ def argparse():
     sys.exit(0)
   ad = {}
   args = Args()
+  # Separate mode selector from other arguments
+  if av[0].startswith("-") and len(av[0]) > 2:
+    av.insert(1, f'-{av[0][2:]}')
+    av[0] = av[0][:2]
   # Support a few other forms for Age etc. compatibility (but only as the first arg)
-  if av[0].lower() in ('enc', 'encrypt', '-e', '--encrypt'):
+  if av[0].lower() in ('enc', 'encrypt', '-e', '--encrypt') or av[0].startswith('-e'):
```

Comment:
Don't need the extra check here. Line 105 already turns it into `-e` which is covered already on the old check.

However, `--encrypt` no longer works, so that option needs to be removed (forcing people to use `enc`, `encrypt` or something beginning with `-e`). Should probably remove the `.lower()` too, so that `-E` or `ENC` won't be accepted here.

Ditto for decrypt options.